packages/components/ng-uirouter-layout/README.md

```diff
@@ -11,6 +11,8 @@ yarn add @ovh-ux/ng-uirouter-layout
 ```
 ## Usage
 
+### In `index.js`
```

Comment:
Prefer specifying the current file by adding first commented line.

```js
// index.js
import angular from 'angular';
import ngUiRouterLayout from '@ovh-ux/ng-uirouter-layout';

angular
  .module('myApp', [
    ngUiRouterLayout,
  ]);
```
Same with the `routing.js` file example.

In addition it would great to provide real examples through the `examples` folder (recently introduced).

What do you think about it?

Thanks!